features-json/apng.json

```
@@ -1,8 +1,7 @@
 {
   "title":"Animated PNG (APNG)",
   "description":"Like animated GIFs, but allowing 24-bit colors and alpha transparency",
-  "spec":"https://wiki.mozilla.org/APNG_Specification",
-  "status":"unoff",
+  "spec":"https://www.w3.org/TR/png",
```

Hi there Chris 👋 

Great that it's part of the spec now! :)

I think the CI expects the `status` to be present, so [from the list](https://github.com/Fyrd/caniuse/blob/main/CONTRIBUTING.md#supported-changes) I think `wd` would be most applicable?

Like so:

```suggestion
  "spec":"https://www.w3.org/TR/png",
  "status":"wd",
```